osal/win32/osal.c

```diff
@@ -130,23 +130,23 @@ void osal_free(void *ptr)
    free(ptr);
 }
 
-int osal_thread_create(void **thandle, int stacksize, void *func, void *param)
+int osal_thread_create(void *thandle, int stacksize, void *func, void *param)
 {
-   *thandle = CreateThread(NULL, stacksize, func, param, 0, NULL);
+   *(void **)thandle = CreateThread(NULL, stacksize, func, param, 0, NULL);
```

Comment:
I'd prefer to typecast it to PHANDLE instead of void ** , very much like the linux port


int osal_thread_create(void *thandle, int stacksize, void *func, void *param)
{
   PHANDLE phandle = thandle;
   
   *phandle = CreateThread(NULL, stacksize, func, param, 0, NULL);
   if(*phandle != NULL)
   {
      return 0;
   }
   return 1;
}

int osal_thread_create_rt(void *thandle, int stacksize, void *func, void *param)
{
   int ret;
   PHANDLE phandle = thandle;
   
   ret = osal_thread_create(phandle, stacksize, func, param);
   if (ret)
   {
      ret = SetThreadPriority(*phandle, THREAD_PRIORITY_TIME_CRITICAL);
   }
   return ret;
}